hx_lti_initializer/views.py

```diff
@@ -213,6 +213,16 @@ def launch_lti(request):
         object_id = get_lti_value(settings.LTI_OBJECT_ID, tool_provider)
         course_id = str(course)
         if set(roles) & set(settings.ADMIN_ROLES):
+            try:
+                userfound = LTICourseAdmin.objects.get(
```

Comment:
@arthurian I would assume that you need a different admin_unique_identifier instead of the username? Or would it work with the username?